slime/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for example.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+configmaps: []
+#- metadata:
+#    name: slime
+#    annotations:
+#      foo: bar
+#  data:
+#    slime: '₍Ꙭ̂₎'
+
+secrets: []
+#- metadata:
+#    name: slime
+#    annotations:
+#      foo: bar
+#  data:
+#    slime: '₍Ꙭ̂₎'
```

Comment:
@cw-sakamoto 
Since values ​​is empty, helm do not create anything.

https://circleci.com/gh/chatwork/charts/517?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
Error: release slime-13b7d13 failed: no objects visited
```

Therefore, CI fails.
What to do for chart?

Reply:
You can configmap and secrets value in Makefile ?
https://github.com/chatwork/charts/blob/13b7d13e39642ca0690a57e08389f9f593e8e093/slime/Makefile#L6

I am annoyed at how to write the test....🤔

Reply:
@cw-sakamoto 
That's good!!

Reply:
good 👍